src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -349,6 +359,16 @@ Loan checkIn(DateTime returnDate, UUID servicePointId) {
     return this;
   }
 
+  public Loan declareItemLost(String comment) {
+    changeAction(DECLARED_LOST_ACTION_NAME);
+    if (StringUtils.isNotBlank(comment)) {
+      changeActionComment(comment);
+    }
```

Comment:
Due to the leaky way action comments are used (as a way of populating the loan history in storage) it might be that a blank comment should remove the action comment in the loan, otherwise the previous comment could appear with the lost declaration line in the UI

Reply:
I tried to implement passing the date from UI. But:
The UI takes the date of an action from the metadata.updatedDate field. Hence, we can't write to it from mod-circulation, can we?
I suppose it is also not a good idea to update the trigger to copy from declaredLostDate field to updatedDate field. What do you think about this?

Reply:
Empty comment does have no effect on previous rows.

Reply:
How have you verified this?

Reply:
yes

Reply:
You recreated the situation locally where a comment had been provided for the immediately previous action, and not for declared lost?

Reply:
Now I got the issue you are refering to. It's been fixed.